docs/stability-tiers.md

```diff
@@ -0,0 +1,194 @@
+  Features are otherwise fully tested on CI.
+
+* Complete implementations for any feature that's part of Tier 1. For example
+  all Tier 2 targets must implement all of the Tier 1 WebAssembly proposals.
+
+* All existing developers are expected to handle minor changes which affect Tier
+  2 components. For example if Cranelift's interfaces change then the developer
+  changing the interface is expected to handle the changes for Tier 2
+  architectures so long as the affected part is relatively minor. Note that if a
+  more substantial change is required to a Tier 2 component then that falls
+  under the next bullet.
+
+* Maintainers of a Tier 2 feature are responsive (reply to requests within a
+  week) and are available to accommodate architectural changes that affect their
+  component. For example more expansive work beyond the previous bullet where
+  contributors can't easily handle changes are expected to be guided or
+  otherwise implemented by Tier 2 maintainers.
+
+* Major changes otherwise requiring an RFC that affect Tier 2 components are
+  required to consult Tier 2 maintainers in the course of the RFC. Major changes
+  to Tier 2 components themselves do not require an RFC, however.
+
+Features at this tier generally are not turned off or disabled for very long.
+Maintainers are already required to be responsive to changes and will be
+notified of any unrelated change which affects their component. It's recommended
+that if a component breaks for one reason or another due to an unrelated change
+that the maintainer either contributes to the PR-in-progress or otherwise has a
+schedule for the implementation of the feature.
+
+#### Tier 1 - Production Ready
+
+This tier is intended to be the highest level of support in Wasmtime for any
+particular feature, indicating that it is suitable for production environments.
+This conveys a high level of confidence in the Wasmtime project about the
+specified features.
+
+Tier 1 features include:
+
+* Continuous fuzzing is required for WebAssembly proposals. This means that any
+  WebAssembly proposal must have support in the `wasm-smith` crate and existing
+  fuzz targets must be running and exercising the new code paths. Where possible
+  differential fuzzing should also be implemented to compare results with other
+  implementations.
+
+* Continuous fuzzing is required for the architecture of supported targets. For
+  example currently there are three x86\_64 targets that are considered Tier 1
+  but only `x86_64-unknown-linux-gnu` is fuzzed.
+
+* CVEs and security releases will be performed as necessary for any bugs found
+  in features.
+
+* Most developers of Wasmtime are expected to be able to handle up-to-major
+  changes affecting this component. Specialized maintainers may still be
+  necessary for major refactorings or major new features, but otherwise it
+  should be reasonable to expect most Wasmtime developers to be able to maintain
+  Tier 1 features.
```

Comment:
I think this rewrite means the same thing but it confuses me less:

```suggestion
* Major changes affecting this component may require help from maintainers with
  specialized expertise, but otherwise it should be reasonable to expect most
  Wasmtime developers to be able to maintain Tier 1 features.
```